org.openscenegraph.android/jni/JNIUtils.h

```diff
@@ -37,6 +38,51 @@ class ReferencedType: public T, public osg::Referenced
 protected:
 	~ReferencedType() { }
 };
+typedef ReferencedType<osg::Vec2> RefVec2;
 typedef ReferencedType<osg::Vec3> RefVec3;
 typedef ReferencedType<osg::Vec4> RefVec4;
 typedef ReferencedType<osg::Quat> RefQuat;
+
+inline osg::Vec4 Interpolate(osg::Vec2 _texcoord, osg::Image* _img)
```

Comment:
shouldn't it be "const osg::Vec2 &_texcoord"? Also we tend to use underscore for member class variables, not for parameters.